js/modal.js

```diff
@@ -64,6 +66,28 @@
     this.resize()
 
     this.$element.on('click.dismiss.bs.modal', '[data-dismiss="modal"]', $.proxy(this.hide, this))
+    this.$element.on('show.bs.collapse hide.bs.collapse', function (e) {
+      var finished = false
+      var rAF      = !!window.requestAnimationFrame
+
+      function adjustDialogPadding() {
+        that.adjustDialog()
+        if (!finished) window.requestAnimationFrame(adjustDialogPadding)
+      }
+
+      $(e.target).one('shown.bs.collapse hidden.bs.collapse', function () {
+        finished = true
```

Comment:
what do you think about renaming `finished` to something like `transitionComplete`?